.github/workflows/test-docker-pulls.yml

```diff
@@ -0,0 +1,41 @@
+---
+# Ensures that all of the default images in our
+# helm values.yaml file are pullable to authenticated
+# users.
+# Triggered on the first PR opening to the main branch.
+# Re-runs will need to be manually kicked off.
+# Add 'synchronize' to `on.pull_request.types`
+# if that's a hassle.
+
+name: Tests - Artifact Pulls
+
+on:
+  pull_request:
+    types:
+      - opened
+    branches:
+      - main # test only on first opening to a main branch
```

Comment:
in the scenario where we are introducing a new image, this test will fail until after the image is published...

Which means we would need to manually run the test again to get it to work?

If there are updates to the PR we don't want to run the tests again?